rest-service/manager_rest/test/endpoints/test_usage_collector_triggers.py

```diff
@@ -0,0 +1,91 @@
+import os
+from datetime import datetime
+
+from manager_rest.storage import models, user_datastore
+from manager_rest.test import base_test
+from integration_tests.tests.utils import (
+    wait_for_deployment_deletion_to_complete
```

Comment:
in these tests, executions aren't actually _ran_, so no need to wait for them

Reply:
Yeah, my bad